library/aws/shared.ts

```diff
@@ -496,7 +496,7 @@ export function createNetworkResources(
       tfgen,
       "vpc_logs_role",
       {
-        name: "vpc_logs_role",
+        name: `${tfgen.nameContext().join("_")}_vpc_logs_role`,
```

Comment:
DRY: extract `${tfgen.nameContext().join("_")}` into `const prefix = tfgen.nameContext().join("_")` and use it.